packages/devtools_app/lib/src/app_size/file_import_container.dart

```diff
@@ -128,29 +129,36 @@ class _FileImportContainerState extends State<FileImportContainer> {
     );
   }
 
-  Widget _buildImportedFileDisplay() {
-    return Text(
-      importedFile?.path ?? 'No File Selected',
-      style: TextStyle(
-        color: Theme.of(context).textTheme.headline1.color,
-      ),
-      textAlign: TextAlign.left,
+  Widget _buildImportedFileDisplay() => Text(
+        importedFile?.path ?? 'No File Selected',
+        style: TextStyle(color: Theme.of(context).textTheme.headline1.color),
+        textAlign: TextAlign.left,
+      );
+
+  Widget _buildImportButton() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        OutlinedButton(
+          onPressed: () async {
+            final acceptedTypeGroups = [
+              XTypeGroup(extensions: ['json']),
+            ];
```

Comment:
can you make `List<String> acceptedTypeGroups` an optional parameter in the `FileImportContainer` constructor that defaults to `['json']`, and use that here instead?